tests/modeladmin/tests.py

```diff
@@ -277,6 +278,47 @@ class BandAdmin(ModelAdmin):
             ['main_band', 'opening_band', 'day', 'id', 'DELETE']
         )
 
+    def test_overriding_get_exclude(self):
+        """
+        Ensure that overriding ModelAdmin.get_exclude() method works.
```

Comment:
I think the docstring isn't adding any value, since it's basically the same as the method name.